### PR TITLE
Move static lists to `/environment` endpoint

### LINKED ISSUE #2301: DRF browsable API returns 500 for `/me` endpoint
Workaround: `/me?format=json` does return the expected response.

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/staticfiles/handlers.py", line 63, in __call__
    return self.application(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/whitenoise/base.py", line 66, in __call__
    return self.application(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/wsgi.py", line 189, in __call__
    response = self.get_response(request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 218, in get_response
    response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 261, in handle_uncaught_exception
    return debug.technical_500_response(request, *exc_info)
  File "/usr/local/lib/python2.7/dist-packages/django_extensions/management/technical_response.py", line 6, in null_technical_500_response
    six.reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 164, in get_response
    response = response.render()
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 158, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/response.py", line 72, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/renderers.py", line 706, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/renderers.py", line 641, in get_context
    raw_data_patch_form = self.get_raw_data_form(data, view, 'PATCH', request)
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/renderers.py", line 559, in get_raw_data_form
    data = {k: v for (k, v) in serializer.data.items()
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/renderers.py", line 560, in <dictcomp>
    if not isinstance(serializer.fields[k],
  File "/usr/local/lib/python2.7/dist-packages/rest_framework/utils/serializer_helpers.py", line 148, in __getitem__
    return self.fields[key]
KeyError: 'available_sectors'
```

### PR DESCRIPTION
**Pull request by jnm**
…including sectors, countries, all known languages, and languages for which the interface has translations. The latter has been renamed from `languages` to `interface_languages`. These were previously included in the `/me` endpoint.

Additional clean-ups:
* Re-sort imports per agreement with @noliveleger
* Remove obsolete downtime-notice code

Fixes #2301

### CODE REVIEW

File: jsapp/js/stores.es6 [hunk 1]
```diff
@@ -219,52 +219,33 @@ var sessionStore = Reflux.createStore({
   },
   triggerAnonymous (/*data*/) {},
   triggerEnv (environment) {
-    this.environment = environment;
-  },
-  triggerLoggedIn (acct) {
-    this.currentAccount = acct;
-    if (acct.upcoming_downtime) {
```
Comment (magicznyleszek) [inline jsapp/js/stores.es6]: Are we dropping all this upcoming downtime code?

Comment (jnm) [reply jsapp/js/stores.es6]: Yes :) if there's any more of it, it should be removed. I think it's been broken for a long time and https://github.com/kobotoolbox/kpi/pull/2260 takes care of its job now

**Comment by noliveleger** [comment]
Even if unittests failed with Travis. They all pass locally with exact same versions of libraries and plugins.

```
platform linux2 -- Python 2.7.15, pytest-3.10.0, py-1.8.0, pluggy-0.12.0 -- /usr/local/bin/python
cachedir: .pytest_cache
Django settings: kobo.settings.testing (from environment variable)
rootdir: /srv/src/kpi, inifile: pytest.ini
plugins: cov-2.7.1, celery-4.2.1, django-3.5.0, env-0.6.2
collected 334 items
...
====================================================================================================================================== 334 passed, 6 warnings in 177.71 seconds == 
```

**Comment by jnm** [comment]
It was failing because the Travis CI environment didn't have translations enabled :)

**Pull request merged**